lib/router.js

```diff
@@ -69,7 +74,7 @@ Cherrytree.prototype.map = function (routes) {
 
   eachBranch({routes: this.routes}, [], function (routes) {
     // concatenate the paths of the list of routes
-    var path = _.reduce(routes, function (memo, r) {
+    var path = routes.reduce(function (memo, r) {
```

Comment:
IIRC the suqcelabs matrix is all es5 but yes do that all together in another PR - I'll revert. I'm sure I accidentally did it anywyas.